collectors/python.d.plugin/mail_log/mail_log.chart.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
+# Description: mail log netdata python.d module
+# Author: apardyl
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
+import re
+from bases.FrameworkServices.LogService import LogService
+
+ORDER = ['incoming_statuses', 'incoming_codes', 'outgoing_statuses', 'outgoing_codes']
+
+CHARTS = {
+    'incoming_statuses': {
+        'options': [None, 'Incoming email statuses', 'messages/s', 'incoming',
+                    'mail_log.incoming_statuses', 'stacked'],
+        'lines': [
+            ['mail_accepted', 'accepted', 'incremental', 1, 1],
+            ['mail_greylisted', 'greylisted', 'incremental', 1, 1],
+            ['mail_temp_rejected', 'temporarily rejected', 'incremental', 1, 1],
+            ['mail_perm_rejected', 'permanently rejected', 'incremental', 1, 1]
+        ]
+    },
+
+    'incoming_codes': {
+        'options': [None, 'Incoming email status codes', 'messages/s', 'incoming',
+                    'mail_log.incoming_codes', 'stacked'],
+        'lines': [
+            ["mail_incoming_codes_2.0.0", '2.0.0', 'incremental', 1, 1]
+        ]
+    },
+
+    'outgoing_statuses': {
+        'options': [None, 'Outgoing email statuses', 'messages/s', 'outgoing',
+                    'mail_log.outgoing_statuses', 'stacked'],
+        'lines': [
+            ['mail_sent', 'sent', 'incremental', 1, 1],
+            ['mail_deferred', 'deferred', 'incremental', 1, 1],
+            ['mail_bounced', 'bounced', 'incremental', 1, 1]
+        ]
+    },
+
+    'outgoing_codes': {
+        'options': [None, 'Outgoing email status codes', 'messages/s', 'outgoing',
+                    'mail_log.outgoing_codes', 'stacked'],
+        'lines': [
+            ["mail_outgoing_codes_2.0.0", '2.0.0', 'incremental', 1, 1]
+        ]
+    },
+}
+
+
+class MailStatistics:
+    # incoming
+    incoming_codes = {
+        "2.0.0": 0
+    }
+    accepted = 0
+    temp_rejected = 0
+    perm_rejected = 0
+    greylisted = 0
+    # outgoing
+    outgoing_codes = {
+        "2.0.0": 0
+    }
+    sent = 0
+    deferred = 0
+    bounced = 0
+
+
+class MailParser:
```

Comment:
You need to have this class inherit from `object`, otherwise things may not work correctly on Python 2.x (fixing this will also get rid of the new alert reported by LGTM).